package.json

```diff
@@ -187,7 +187,10 @@
 		"calypso-codemods": "workspace:^",
 		"config": "^3.3.6",
 		"debug": "^4.3.3",
+		"dom-serializer": "^2.0.0",
+		"domutils": "^3.0.1",
 		"enhanced-resolve": "5.9.3",
+		"htmlparser2": "^8.0.1",
```

Comment:
`htmlparser2`, `domutils`, `dom-serializer` are needed for the server-friendly DOMParser equivalent for the plugin content sanitizer.